.github/workflows/checklinks.yml

```diff
@@ -0,0 +1,13 @@
+name: Check Markdown links
+
+on:
+  schedule:
+  - cron: "0 13 * * 1"
+
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: gaurav-nelson/github-action-markdown-link-check@v1
+      continue-on-error: true
```

Comment:
I couldn't see this option in the docs? https://github.com/gaurav-nelson/github-action-markdown-link-check#custom-variables

Reply:
@eddiejaoude most of the PR is based on this repo, that's why this line exists
- https://github.com/gaurav-nelson/github-action-markdown-link-check

## Preview
![image](https://user-images.githubusercontent.com/18606136/191464616-55bf8aa9-2adf-47cf-b5b2-5e8287820f83.png)

Reply:
@eddiejaoude 
You're right, it's from this article
- https://dev.to/thibaultduponchelle/check-markdown-links-with-github-action-3c2m

## Preview
![image](https://user-images.githubusercontent.com/18606136/191465637-08e545a2-e970-4234-88a2-8f4c53e46e52.png)

Reply:
Not sure if it is still available on the Action, might need to dig deeper.

If it is available, it is probably best for @EmmaDawsonDev to decide if she prefers failure or report only